tests/dummy/config/targets.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-env node */
-const browsers = ['last 1 Chrome versions', 'last 1 Firefox versions', 'last 1 Safari versions'];
+const browsers = ['ie 11', 'last 1 Chrome versions', 'last 1 Firefox versions', 'last 1 Safari versions'];
```

Comment:
We should probably revert this change (to test locally you can use `CI=true ember test --server` or `CI=true ember s -p 0`)...